.github/workflows/rust_test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: install foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
-          version: nightly
+          version: nightly-a79dfaed6fc6f88cda5f314a25d1b484d9d8c051
```

Comment:
This should not be necessary anymore 
The issue is closed and should be fixed normally https://github.com/foundry-rs/foundry/issues/9349
cc @enitrat